.github/workflows/tests.yaml

```diff
@@ -16,8 +16,8 @@ jobs:
         dependencies: ['']
         xdebug-ini-values: ['']
         include:
-          - { php-versions: '7.1', xdebug-ini-values: 'xdebug.overload_var_dump=0' }
-          - { php-versions: '7.2', xdebug-ini-values: 'xdebug.overload_var_dump=0' }
+          - { php-versions: '7.1', xdebug-ini-values: 'xdebug.mode=coverage' }
+          - { php-versions: '7.2', xdebug-ini-values: 'xdebug.mode=coverage' }
```

Comment:
Could you please remove this commit? I independently meanwhile adjusted in in a different way in main branch.

Reply:
Sure thing.